Frameworks/UIKit/XamlUtilities.mm

```diff
@@ -13,17 +13,16 @@
 // THE SOFTWARE.
 //
 //******************************************************************************
-
 #import "XamlUtilities.h"
-#import "UIViewInternal+Xaml.h"
-#import "CACompositor.h"
+#import "StarboardXaml/DisplayTexture.h"
+
+#include <Windows.UI.Xaml.Media.h>
 
 using namespace Microsoft::WRL;
 using namespace Windows::Foundation;
 
 // cornerRadius when border style is set to round rectangle
 static const int c_borderCornerRadius = 8;
-static const wchar_t* TAG = L"XamlUtilities";
```

Comment:
it's not used, but I can add it back for the next person who needs to add logging to this file.

---
In reply to: [86710622](https://github.com/Microsoft/WinObjC/pull/1318#discussion_r86710622) [](ancestors = 86710622)